src/EzPlatformCoreBundle/bundle/DependencyInjection/Compiler/SessionConfigurationPass.php

```diff
@@ -54,12 +54,21 @@ public function process(ContainerBuilder $container)
                 $container->setAlias('session.handler', $handlerId);
             }
 
-            $container
-                ->getDefinition('session.storage.native')
-                ->replaceArgument(1, new Reference('session.handler'));
-            $container
-                ->getDefinition('session.storage.php_bridge')
-                ->replaceArgument(0, new Reference('session.handler'));
+            if ($container->hasDefinition('session.storage.native')) {
+                $container
+                    ->getDefinition('session.storage.native')
+                    ->replaceArgument(1, new Reference('session.handler'));
+                $container
+                    ->getDefinition('session.storage.php_bridge')
+                    ->replaceArgument(0, new Reference('session.handler'));
+            } else {
+                $container
+                    ->getDefinition('session.storage.factory.native')
+                    ->replaceArgument(1, new Reference('session.handler'));
+                $container
+                    ->getDefinition('session.storage.factory.php_bridge')
+                    ->replaceArgument(0, new Reference('session.handler'));
+            }
```

Comment:
We should check for each & every service separately, and replace argument in each instance.

This way we avoid the assumption that when `session.storage.native` exists, there will also be a `session.storage.php_bridge`. And if somehow all 4 exist, then it will apply to all 4, not just 2.

Reply:
It's just a copy-paste from Symfony FrameworkExtension but I see your point. I will change it soon.

Reply:
Fixed: db17482